js/app.js

```diff
@@ -8,23 +8,17 @@ var imageA = '';
 var imageB = '';
 var imageC = '';
 var productObject = {}; // object to store the product objects
-// timesClicked; // number of times a product has been clicked
-// timesShown; // number of times a product has been shown
 var currentImages = []; // current products array
 var previousImages = []; // previous products array
-// var previousImages = ['bag', 'chair', 'water-can'];
 var userClicks = 0; // number of times the user has clicked
 var maxClicks = 24; // total number of clicks the user is allowed
 var imagesParent = document.getElementById('images'); // parent element on index where the images will be displayed
-// var resultsList = document.creteElement('ul'); // create list element to display results list on index
+var resultsList = document.createElement('ul'); // create list element to display results list on index
 
 
 // ================================
 // ========CONSTRUCTOR=============
 // ================================
-// * Have a product object constructor that contains:
-// * Product name // * image // * Times clicked // * Times shown
-
 function Product (name) {
   this.name = name;
   this.image = 'img/' + name + '.jpg';
```

Comment:
so...something like this.image = a call to a function that does this and returns the proper name for the image like I did in the set attribute part?